### PR TITLE
Collapse logs and lists with HTML5 support of github

### PR DESCRIPTION
**Pull request by nickvergessen**
Should help making our long reports a bit more visual:
https://twitter.com/ericclemmons/status/749223563790471169

@MorrisJobke @LukasReschke

### CODE REVIEW

**Comment by LukasReschke** [comment]
Works! 🚀 👍

File: issue_template.md [hunk 1]
```diff
@@ -90,20 +102,29 @@ Eventually replace sensitive data as the name/IP-address of your LDAP server or
...
+<details>
+<summary>Browser log</summary>
 ```
 Insert your webserver log here
 ```
+</details>
 
 #### Nextcloud log (data/owncloud.log)
+<details>
+<summary>Browser log</summary>
```
Comment (MorrisJobke) [inline issue_template.md]: Nextcloud log ;)

File: issue_template.md [hunk 2]
```diff
@@ -90,20 +102,29 @@ Eventually replace sensitive data as the name/IP-address of your LDAP server or
 
 ### Logs
 #### Web server error log
+<details>
+<summary>Browser log</summary>
```
Comment (MorrisJobke) [inline issue_template.md]: Web server error log

**Comment by MorrisJobke** [comment]
I fixed this and works (see #297 ) 👍

**Pull request merged**